contracts/modules/social_recovery/SocialRecoveryModule.sol

```diff
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: GPL-3.0
```

Comment:
Is this some weird merge thing causing additional changes in the PR?

Reply:
https://github.com/5afe/CandideWalletContracts/pull/32#issuecomment-2154574625